scripts/scripts.js

```diff
@@ -939,7 +939,7 @@ function computeTaxonomyFromTopics(topics, path) {
     const visibleTopics = [];
     // if taxonomy loaded, we can compute more
     topics.forEach((tag) => {
-      const tax = taxonomy.get(tag);
+      const tax = taxonomy.get(tag.trim());
```

Comment:
I found a small bug that all the tag except the first one has a whitespace in front of the tag value. I added trim() to resolve the bug.